src/Okta.Idx.Sdk/IIdxResponse.cs

```diff
@@ -52,6 +52,11 @@ public interface IIdxResponse : IResource
         /// <returns>An IdxResponse.</returns>
         Task<IIdxResponse> CancelAsync(CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Gets the current authenticator
+        /// </summary>
+        IAuthenticatorEnrollmentData CurrentAuthenticator { get; }
```

Comment:
What's the diff between this prop and `CurrentAuthenticatorEnrollment`? they both seem to be the same type?

Reply:
They are of the same type but names are different, one is being returned when one-step-auth is configured and another for two-steps-auth

Reply:
they are used for password recovery flow